src/transport.cpp

```diff
@@ -94,7 +94,11 @@ namespace quicr {
         spdlog::drop(logger_->name());
     }
 
-    void Transport::Init() {}
+    void Transport::Init()
+    {
+        ctrl_serial_buffer_.reserve(4096);
+        serial_buffer_.reserve(8192);
```

Comment:
Considering it'll grow, it's probably okay not to be in the transport config, but it would be nice to have it as a consexpr.